### PR TITLE
fix various errors (404 and float/string)

### PR DESCRIPTION
**Pull request by azet**

### CODE REVIEW

File: jessie.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
             "headless": "true",
             "disk_size": "{{ user `disk_size`}}",
             "qemuargs": [[ "-m", "1024M" ],[ "-smp", "4" ]],
-            "iso_url": "http://cdimage.debian.org/cdimage/jessie_di_rc1/amd64/iso-cd/debian-jessie-DI-rc1-amd64-netinst.iso",
-            "iso_checksum": "2ab639aa971279f2cfbe45ea01a51529",
+            "iso_url": "http://debian.anexia.at/debian-cd/current/amd64/iso-cd/debian-8.2.0-amd64-netinst.iso",
```
Comment (sammcj) [inline jessie.json]: This should probably still be cdimage.debian.org which will select the closest mirror to the user

**Comment by sammcj** [comment]
Thanks for the PR - I haven't used this in ages actually (well, since RC1 funnily enough!), if you fix that one little thing with the mirror I'll merge.

**Comment by azet** [comment]
@sammcj fixed in commit 6e658650f4001ff61b6b0136c4e029171e6fe617.

**Comment by azet** [comment]
:+1:

**Pull request merged**